.flowconfig

```diff
@@ -29,6 +29,12 @@
 
 
 [include]
+; Effectively this line warns Flow that this path might be a symlink
+; (because the developer might be using `yarn link` to have this point into
+; their zulip.git worktree), and it should watch/inotify at the target.
+node_modules/@zulip/shared/
+; Add more as needed!  See also metro.config.js .
```

Comment:
This (and the comment in `metro.config.js`, as well) should probably point "up" to the `yarn-link` document, rather than "sideways" to a sibling.

(Also, the space before the period shouldn't be necessary.)

Reply:
> This (and the comment in `metro.config.js`, as well) should probably point "up" to the `yarn-link` document, rather than "sideways" to a sibling.

Sure, good idea -- that also gives an answer to the question of where to link to that doc from. (As it stands, there are no links to it other than browsing the tree, and in our heads.)

I think the "sibling" link is actually still helpful, though: if you have the general idea and just need a reminder of the specifics, that is a lot fewer words (about 400 fewer) than the doc.

> (Also, the space before the period shouldn't be necessary.)

I generally prefer to minimize ambiguity about where the name ends and the surrounding sentence's punctuation begins.

In this example, if in my terminal I have this sentence without that space, then double-clicking on the filename selects (and puts in the clipboard for middle-click paste) the whole "word" including the period -- IOW, the ambiguity is enough to fool the terminal's heuristic for conveniently selecting and copying in units of words.

Reply:
> double-clicking on the filename

I suspected; and I _thought_ I'd tested exactly that and had it select the filename without the period – but now I can't reproduce it. `>_<` Let me retract that, and instead suggest ``` `metro.config.js` ``` (or some other quotative mark), just so it doesn't look like a typo.

(But it's not important.)